### PR TITLE
Feature/stack ns

### PR DESCRIPTION
**Pull request by andre-merzky**
generalize the stack query to support other name spaces than just `radical`.

Alas, this PR also removes the `-v` flag from `radical-stack`, as pip does not support the respective query anymore.  We need to look for an alternative way to implement this (related to radical-cybertools/radical.entk/issues/476)

### CODE REVIEW

File: src/radical/utils/stack.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                mbase = os.path.basename(mpath)
+                mname = '%s.%s' % (namespace, mbase)
+
+                if mbase.startswith('_'):
+                    continue
+
+                try:
+                    ret[namespace][mname] = import_module(mname).version_detail
+                except Exception as e:
+                    if 'RADICAL_DEBUG' in os.environ:
```
Comment (mtitov) [inline src/radical/utils/stack.py:57]: should env var also be updated? `'RADICAL_DEBUG'` -> `'%s_DEBUG' % namespace.upper()`

Comment (andre-merzky) [reply src/radical/utils/stack.py]: Thanks, makes sense.

**Review by mtitov — APPROVED** [review]
LGTM

**Comment by andre-merzky** [comment]
Thanks, appreciated! :-)

**Pull request merged**